lib/providers/imageprovider.dart

```diff
@@ -33,6 +32,9 @@ class InlineImageProvider extends ChangeNotifier {
   //else delte operation is performed.
   int controllerLength = 0;
 
+  //Map to store the hex values of the characters
+  final Map<int, List<String>> hexStrings = {};
```

Comment:
Unused?

Reply:
yes I created this because I was thinking that I should cache the hex values of the vectors that the user clicks maybe that could be helpful.

Reply:
so that when the user selects the same vector again the value will be directly fetched by the map.

Reply:
You image provider should handle that. Wrap the images inside this class which caches the image object and the cache for hex

Reply:
ok I will update it in the next PR

Reply:
Sure, can you remove it for now and the delay added